pkg/option/config.go

```diff
@@ -331,6 +331,9 @@ const (
 	// LogOpt sets log driver options for cilium
 	LogOpt = "log-opt"
 
+	// EnableRemoteNodeMasquerade Masquerade packets from endpoints leaving the host destined to a remote node node in BPF masquerading mode. This option requires to set enable-bpf-masquerade to true.
```

Comment:
A few typos here (i.e. "node node"). Suggested description:

"Masquerade packets from endpoints leaving the host destined for a remote node in BPF masquerading mode. This option requires that enable-bpf-masquerade is set to true."